### PR TITLE
April deployment

### PR DESCRIPTION
**Pull request by christianrodriguez-8451**
Here are a bunch of changes that I've been sitting on! I am pushing them now since we are deploying about 30 segments to PRISM.

### CODE REVIEW

**Review by stevenmartz-8451 — CHANGES_REQUESTED** [review]
Funlo methodology uses these date range weights so we need to keep these in the config:  DATE_RANGE_WEIGHTS = {4: 0.27, 3: 0.26, 2: 0.24, 1: 0.23}

**Review by stevenmartz-8451 — APPROVED** [review]

File: toolbox/egress_prism_input_data.py
Comment (stevenmartz-8451) [inline toolbox/egress_prism_input_data.py:1]: Recently Steve Ruwe pulled me into an issue where an audience wasn't meeting a Prism household minimum of 2 million.  We need to check with Brian Newell and team if we can set our threshold this low.  I know we keep getting conflicting information but I don't want us to cause user issue.

**Review by stevenmartz-8451 — CHANGES_REQUESTED** [review]
Please see my comments on products_embedding/config.py and toolbox/egress_prism_input_data.py.

File: toolbox/egress_prism_input_data.py
Comment (christianrodriguez-8451) [reply toolbox/egress_prism_input_data.py:1]: Is this threshold applied at audience or campaign level? What is the real threshold for an audience? For now, I set it back to 2 million. But call out on my end - a lot of the sensitive audiences are well below the 2 million count.

Comment (stevenmartz-8451) [reply toolbox/egress_prism_input_data.py:1]: I'm pretty sure it's at a campaign level.  Meaning if they choose one audience that's 1.5 MM then they will run into issues with the campaign.  But if they choose a 1.5 MM and a 2 MM audience giving them 3.5 MM for the campaign, they are fine.  I think Prism should check for this and notify the user.  I don't think our audiences should have that high of a limit.

**Review by stevenmartz-8451 — APPROVED** [review]
A lot of work has been done!  :)

**Pull request merged**